### PR TITLE
React basic config

### PR DESCRIPTION
**Pull request by fkrugerr**

### CODE REVIEW

**Comment by sergeykuzmich** [comment]
@artembasov, @skolola-rps, please, review this basic code style configuration for react/react-native projects. 
@skolola-rps, schedule a call to discuss it if needed, and then give it to the others to enable it in their projects.

**Comment by fkrugerr** [comment]
> What about typescript config?
In the same directory? I think we need a separate folder for TS, e.g react-ts.

**Comment by sergeykuzmich** [comment]
You need to decide do you actually need TypeScript.

Some times development becomes the war between libraries (that does not support typescript) & typescript instead of just development. A lot of `: any` says that in the most projects we don't need typescript at all.

But anyway, you decide.

**Comment by fkrugerr** [comment]
We  are not obliged to use TS in each project. But we can't avoid it at all. So I will make ts-config in subdirectory 'typescript'.

File: react/typescript/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "react",
+    "baseUrl": "src"
+  },
```
Comment (artembasov) [inline react/typescript/tsconfig.json:18]: I use `src` alias So I set 
`
"baseUrl": ".",
"paths": {
      "src/*": [
        "src/*"
      ],
    },
`
without `include` property.

Comment (fkrugerr) [reply react/typescript/tsconfig.json:18]: It doesn't work in CRA without ejection.

Comment (artembasov) [reply react/typescript/tsconfig.json:18]: Sorry, cra says aliases are not supported

**Pull request merged**